### ISSUE TITLE
Connect to different broker in a multi-broker ActiveMQ instance

### ISSUE THREAD

**Issue by mauromol** [jmstoolbox/jmstoolbox#88]
## Specifications
- JMSToolBox version: 5.8.0
- OS (Name, version, 32/64bits): Windows 10 64-bit
- Q provider (kind, version) : ActiveMQ 5.7.0 in Service Mix

## Expected Behavior
I can successfully connect to the ActiveMQ instance in my Service Mix bundle, but only the "default" broken contents is shown. I can't find a way to select another broker.

## Actual Behavior
I can select the broker to browse, either from within the session or by creating different sessions for each broker.

## Steps to Reproduce the Problem

  1. configure your ActiveMQ with different brokers on different ports (61616, 61615, etc.)
  1. connect to it with JMSToolBox and JMX connection parameters
  1. only the default broker is shown

With JMX I see this, I would like to be able to choose the broker in JMSToolBox too:
![immagine](https://user-images.githubusercontent.com/1742494/92465139-79f6dd00-f1ce-11ea-9cb7-4b77dc44df12.png)
I would like to connect to broker5-prod, for instance, but JMSToolBox just connects to the default one.

**Comment by titou10titou10** [comment]
JMSToolBox uses the JMX connector to connect to the ActiveMQ instance
It is possible to specify the brokername in the MBean name
Currently JTB uses the "first" broker it finds
I will add the parameter for brokerName in JTB
Ref: https://activemq.apache.org/jmx.html

However you are using an old version of ActiveMQ and MBeans have been reorganized after v5.8.0. There are two "path" in JTB to handle the situation. Not sure if it will work with ActiveMQ v5.8.0-

**Comment by mauromol** [comment]
Yes, I'm using an old version because this is the ActiveMQ bundled in an old Service Mix version used in production. If you could make it work with 5.8.0- it would be much appreciated.

**Comment by titou10titou10** [comment]
You wrote that you started 2 brokers on different ports (61616, 61615). 
When you connect to the first or second one (by changing the brokerURL in the session definition in JTB), you always see the list of destinations for the "default" one? whatever the port you use?

**Comment by mauromol** [comment]
I always see the destinations for the default broker whatever port I specify in the brokerURL in JTB.
